src/BlazorSignalR.JS/src/ServerSentEventsTransport.ts

```diff
@@ -15,21 +15,21 @@ export class ServerSentEventsTransport {
         this.connections.set(id, eventSource);
 
         eventSource.onmessage = (e: MessageEvent) => {
-            managedObj.invokeMethod<void>("HandleSSEMessage", btoa(e.data));
+            managedObj.invokeMethodAsync<void>("HandleSSEMessage", btoa(e.data));
```

Comment:
Should also probably be awaited.

Reply:
This is done. The complete callback is an async function now.